mlflow/pytorch/__init__.py

```diff
@@ -128,6 +128,13 @@ def log_model(
     """
     Log a PyTorch model as an MLflow artifact for the current run.
 
+        .. warning::
```

Comment:
Just an FYI -- typically in tech writing, "warning" is used only for serious situations, such as physical injury or data loss. I think it's probably actually appropriate here, since there is a possibility of causing the model to produce inaccurate results.

Reply:
Agreed, @andreakress I would prefer that we use a warning class warning here. This kind of error took me multiple hours to debug and I have a full understanding of our entire stack. Our users could really get turned around here if we don't offer a warning.

Reply:
Ack.